### ISSUE TITLE
Sorting: Return computed values that the sort was performed on

### ISSUE THREAD

**Issue by JirkaChadima** [windingtree/wt-search-api#21]
We are providing a `sortByLocation` option in the search API and will eventually provide more sorting criteria. The thing is that they might operate on computed values (such as distance on Earth between a centerpoint and a hotel location). This of course does not apply to static data such as hotel names which can be easily retrieved from base hotel data.

But when the clients want to show the particular value that hotels were sorted by, they cannot, because the API does not return the computed values, only the base hotel data.

My suggestion is to return the values used for sorting along the hotel data. Either directly with every single hotel, or to keep the consistency with read-api, in a separate structure, something like

```
{
  "items": [...],
  "errors": [...],
  "sorting": [
    {
      "id": "0x...",
      "sorts": [
        {
          "key": "sortByLocation",
          "value": 123
        }
      ]
    }
  ]
}
```

**Comment by hynek-urban** [comment]
One thing that comes to mind with respect to this is that what it's sorted by does not have to make sense for end users to see.

In the case of sortByLocation, it sorts by the square of distance that is expressed in "scaled degrees". To arrive at kilometers, another computation needs to be done - possibly, it might even be easier and more efficient to re-compute it "on the side", i.e. not take it from the database sorting at all.

**Comment by hynek-urban** [comment]
This has been implemented in #22.
